apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -1,16 +1,29 @@
 import { FormProvider } from 'react-hook-form';
-import { Group, Container } from '@mantine/core';
+import { Container, Group } from '@mantine/core';
+import { useState } from 'react';
+import { ChannelTypeEnum } from '@notifire/shared';
 import { useParams } from 'react-router-dom';
 import PageContainer from '../../../components/layout/components/PageContainer';
 import PageHeader from '../../../components/layout/components/PageHeader';
 import { TemplatesSideBar } from '../../../components/templates/TemplatesSideBar';
 import { NotificationSettingsForm } from '../../../components/templates/NotificationSettingsForm';
 import { useTemplateController } from '../../../legacy/pages/templates/editor/use-template-controller.hook';
 import { TemplateTriggerModal } from '../../../components/templates/TemplateTriggerModal';
+import { AddChannelsPage } from './AddChannelsPage';
 import { Button } from '../../../design-system';
 
 export default function TemplateEditorPage() {
   const { templateId = '' } = useParams<{ templateId: string }>();
+  const [activePage, setActivePage] = useState<string>('Settings');
+  const [channelButtons, setChannelButtons] = useState<string[]>([]);
+
+  const handleAddChannel = (tabKey) => {
```

Comment:
Can we make that after we add a channel we switch to the newly created channel instead of staying in the add channel screen?